src/Command/App/AppBranchesTagsInfo.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Acquia\Cli\Command\App;
+
+use Acquia\Cli\Command\CommandBase;
+use AcquiaCloudApi\Endpoints\Code;
+use AcquiaCloudApi\Endpoints\Environments;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class AppBranchesTagsInfo.
+ */
+class AppBranchesTagsInfo extends CommandBase {
+
+  protected static $defaultName = 'app:get-branches-tags';
```

Comment:
Let's say "list" instead of "get" and use colons instead of dashes. Also "branches-tags" is a little awkward.

Maybe `app:list:vcs`? `app:vcs:refs`? `app:list:deployments`?

@anavarre opinions?

Reply:
If we're looking for purity, the closest to that is `app:vcs:refs`. In plain english it means I want to display the version control system references that are deployed or not for my application, which is exactly what the command is doing.

Reply:
I like `app:vcs:refs`. @joshirohit100 would this be intuitive to you as a developer?

Reply:
To me as dev something like `app:vcs:info` is more intuitive.

Reply:
Updated the PR with the command name `app:vcs:info`. Added two more tests.

Reply:
I can get behind `app:vcs:info`